docs/gcloud-auth.rst

```diff
@@ -58,15 +58,66 @@ properties of your local environment in the following order
 #. Credentials provided by the Google Cloud SDK (via ``gcloud auth login``)
 #. Application running in Google Compute Engine
 
-Loading Credentials Explicitly
-------------------------------
+Explicit Credentials
+====================
 
-In addition, the
+The Application Default Credentials discussed above can be useful
```

Comment:
Re-did each line with newlines after (most) commas and periods. Refrained from doing it in situations like `However,` (I think only came up twice).